listener/avalon_listener/tcs_work_order_handler.py

```diff
@@ -206,6 +206,11 @@ def WorkOrderSubmit(self, **params):
               as defined in EEA spec 6.1.1
         Returns jrpc response as defined in EEA spec 6.1.3
         """
+        if "workOrderId" not in params:
+            raise JSONRPCDispatchException(
+                WorkOrderStatus.INVALID_PARAMETER_FORMAT_OR_VALUE,
+                "Missing work order id"
+            )
```

Comment:
Need a similar check in https://github.com/hyperledger/avalon/blob/master/listener/avalon_listener/tcs_work_order_handler_sync.py#L65. 
But rather we should have a common function for both files

Reply:
This could be fixed using jsonschema as well.

Reply:
@manojgop Removed since json-schema address it.